### ISSUE TITLE
"IndexError: pop from empty list" when processing valid file

### ISSUE THREAD

**Issue by chriskuehl** [asottile/add-trailing-comma#3]
(Wasn't sure how to describe this scenario better, sorry for the lame title.)

Here's a fairly minimal example:

```python
(
    a
).thing(b)
```

Produces an error when processing with add-trailing-comma v0.2.0:

```python
Traceback (most recent call last):
  File "/nail/home/ckuehl/.pre-commit/repojgy42wl2/py_env-python2.7/bin/add-trailing-comma", line 9, in <module>
    load_entry_point('add-trailing-comma==0.2.0', 'console_scripts', 'add-trailing-comma')()
  File "/nail/home/ckuehl/.pre-commit/repojgy42wl2/py_env-python2.7/local/lib/python2.7/site-packages/add_trailing_comma.py", line 269, in main
    ret |= fix_file(filename, args)
  File "/nail/home/ckuehl/.pre-commit/repojgy42wl2/py_env-python2.7/local/lib/python2.7/site-packages/add_trailing_comma.py", line 249, in fix_file
    contents_text = _fix_commas(contents_text, args.py35_plus)
  File "/nail/home/ckuehl/.pre-commit/repojgy42wl2/py_env-python2.7/local/lib/python2.7/site-packages/add_trailing_comma.py", line 232, in _fix_commas
    _fix_call(call, i, tokens)
  File "/nail/home/ckuehl/.pre-commit/repojgy42wl2/py_env-python2.7/local/lib/python2.7/site-packages/add_trailing_comma.py", line 189, in _fix_call
    paren_stack.pop()
IndexError: pop from empty list
```

**Comment by asottile** [comment]
neat, I think I know what causes this too

**Comment by asottile** [comment]
After fixing, I also verified it rewrites this file (not that I'd expect it to fail on it)

```diff
 (
     a
 ).f(
-     b
+     b,
 )
```

**Comment by asottile** [comment]
This is available in 0.2.1
